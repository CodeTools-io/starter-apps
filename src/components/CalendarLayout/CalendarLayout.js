import React from 'react';
import { Box, Grid } from 'grommet';

export default function CalendarLayout({ children, ...props }) {
  return (
    <Box className="CalendarLayout" fill {...props}>
      <Grid
        rows={['xsmall', 'flex', 'auto']}
        columns={['1/4', '1/4', '1/4', '1/4']}
        areas={[
          ['search', 'header', 'header', 'header'],
          ['sidebar', 'main', 'main', 'main'],
          ['compose', 'footer', 'footer', 'footer'],
        ]}
        fill
      >
        {children}
      </Grid>
    </Box>
  );
}
