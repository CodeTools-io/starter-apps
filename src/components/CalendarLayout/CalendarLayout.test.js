import React from 'react'
import CalendarLayout from './CalendarLayout'

describe('CalendarLayout', () => {
  test('can render', () => {
    const { container } = renderComponent(
      <CalendarLayout>
        <p>Content for the CalendarLayout</p>
      </CalendarLayout>
    )

    expect(container).toMatchSnapshot()
  })
})

