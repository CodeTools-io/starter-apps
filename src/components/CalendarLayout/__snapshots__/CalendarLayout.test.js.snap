// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarLayout can render 1`] = `
<div>
  <div
    class="StyledBox-sc-13pk1d4-0 iNKrsm CalendarLayout"
  >
    <div
      class="StyledGrid-sc-1wofa1l-0 cIFrny"
    >
      <p>
        Content for the CalendarLayout
      </p>
    </div>
  </div>
</div>
`;
